### PR TITLE
[TV/AnimationSample] Replace ClickEvent by Clicked.

### PR DESCRIPTION
**Pull request by AchoWang**
ClickEvent is deprecated in API8,   and will be removed in API10.  So use Clicked instead!

### CODE REVIEW

File: TV/FlexContainerSample/FlexContainerSample/src/FlexContainerSample.cs [hunk 1]
```diff
@@ -73,9 +71,9 @@ public void Initialize()
             guide.PositionUsesPivotPoint = true;
             guide.ParentOrigin = ParentOrigin.TopLeft;
             guide.PivotPoint = PivotPoint.TopLeft;
-            guide.Size2D = new Size2D(1920, 96);
+            guide.Size = new Size(1920, 96);
             guide.FontFamily = "Samsung One 600";
-            guide.Position2D = new Position2D(0, 94);
+            guide.Position = new Position(0, 94);
             guide.MultiLine = false;
             //guide.PointSize = 15.0f;
```
Comment (dongsug-song) [inline TV/FlexContainerSample/FlexContainerSample/src/FlexContainerSample.cs:78]: please remove useless commented line

File: TV/FlexContainerSample/FlexContainerSample/src/FlexContainerSample.cs [hunk 2]
```diff
@@ -85,15 +83,23 @@ public void Initialize()
...
-            flexContainer.SizeWidth = 400;
-            flexContainer.SizeHeight = 400;
-            flexContainer.BackgroundColor = Color.White;
-            flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Column;
+            flexContainer = new View()
+            {
+                PositionUsesPivotPoint = true,
+                PivotPoint = PivotPoint.TopLeft,
+                ParentOrigin = ParentOrigin.TopLeft,
+                Position = new Position(710, 275, 0),
```
Comment (dongsug-song) [inline TV/FlexContainerSample/FlexContainerSample/src/FlexContainerSample.cs:91]: isn't it "Position(710, 275);" ?

**Comment by dongsug-song** [comment]
@AchoWang 
Hello,
as you see, this patch was reviewed but  was merged without modification.
if you don't mind, could you upload patch again by fixing reviewed codes?
actually, I got warning from SE team that this had been merged without review approved.
please, commit the patch again.
thank you.

**Comment by AchoWang** [comment]
hello, Mr. dongsug-song:
I have fixed above issues in the patch https://github.com/Samsung/Tizen-CSharp-Samples/pull/171, please check it, thanks~

**Comment by myroot** [comment]
@AchoWang  Please rollback this PR, It occurred build error

**Pull request merged**